internal/services/shared/schema.go

```diff
@@ -38,7 +38,9 @@ func EnsureNoRelationshipsExist(ctx context.Context, ds datastore.Datastore, nam
 		return err
 	}
 
-	qy, qyErr = ds.ReverseQueryTuplesFromSubjectNamespace(namespaceName, syncRevision).Limit(1).Execute(ctx)
+	qy, qyErr = ds.ReverseQueryTuples(ctx, &v1.SubjectFilter{
```

Comment:
Do we want to rename `syncRevision` to `headRevision` to match the other commit?